jans-auth-server/client/src/test/java/io/jans/as/client/client/Asserter.java

```diff
@@ -51,15 +51,21 @@ public static void assertOk(RegisterResponse response) {
         assertEquals(response.getStatus(), 201, "Unexpected response code: " + response.getEntity());
         assertNotNull(response.getClientId());
         assertNotNull(response.getClientSecret());
-        assertNotNull(response.getClientIdIssuedAt());
         assertNotNull(response.getClientSecretExpiresAt());
+        assertNotNull(response.getRegistrationAccessToken());
```

Comment:
I guess we can add also assert for `registrationClientUri` as well.

Reply:
assertNotNull(response.getRegistrationClientUri());
is not used in most test cases, due to it I preferred do not include it in **assertOk**
I preserved this assert in cases that it was present.

Reply:
I see, lets unify it. Add assert for registration access token.

Reply:
next asserts was added:
assertNotNull(response.getRegistrationClientUri()); assertNotNull(response.getClientIdIssuedAt());